Makefile

```diff
@@ -90,14 +92,19 @@ cluster-down:
 	./cluster/down.sh
 
 cluster-clean:
-	./cluster/clean.sh
+	$(KUBECTL) delete --ignore-not-found -f build/_output/
+	$(KUBECTL) delete --ignore-not-found -f deploy/
+	$(KUBECTL) delete --ignore-not-found -f deploy/crds/nmstate_v1_nodenetworkstate_crd.yaml
+	if [[ "$$KUBEVIRT_PROVIDER" =~ ^os-.*$$ ]]; then \
```

Comment:
Personally i prefer not to write shell constructs in Makefiles, once we find out we need a little more, it will be an headache. However, I will hope you will be the one that will suffer this and will not fight against this.

Reply:
Agree, the idea is to move things to cluster/ hacks/ when we have a clear view of what we need, but right now is just easier to have it here.